.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+The following is a set of guidelines for contributing to Hey-UI. Please spend several minutes reading these guidelines before you create an issue or pull request.
+
+## Code of Conduct
+
+We have adopted a [Code of Conduct](https://github.com/hey-car/hey-ui/blob/main/CODE_OF_CONDUCT.md) that we expect project participants to adhere to
+elso please read our [Structure Convention](https://github.com/hey-car/hey-ui/blob/main/STRUCTURE_CONVENTION.md) and [Style Guide](https://github.com/hey-car/hey-ui/blob/main/STYLEGUIDE_CONVENTION.).
+Please read the full text so that you can understand what actions will and will not be tolerated.
+
+## Open Development
+
+All work on Hey-UI happens directly on [GitHub](https://github.com/hey-car/hey-ui). Both core team members and external contributors send pull requests which go through the same review process.
+
+## Branch Organization
+
+According to our [release schedule](changelog#Release-Schedule), we maintain two branches, `master` and `feature`. If you send a bugfix pull request, please do it against the `master` branch, if it's a feature pull request, please do it against the `feature` branch.
```

Comment:
our `master` is called `main`, no? unless you changed that, please fix the two `master` mentions